lib/crypto/index.js

```diff
@@ -897,34 +906,40 @@ Crypto.prototype.decryptEvent = function(event) {
  * @param {string} roomId   The ID of the room we want to remind them about.
  */
 Crypto.prototype._sendPingToDevice = function(userId, deviceId, roomId) {
-    if (deviceId === undefined) {
-        deviceId = "*";
```

Comment:
deviceId is always included now, so it just felt like cruft to me. I suppose it might as well stay in.